examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,240 @@
+# Steps to to deploy aurora rds cluster
+This example deploys an aurora postgresql database cluster when applied the claim. Below are the steps to execute to
+create a cluster and connect the cluster from a pod with an psql client.
+
+## Pre-requisites
+ - [Upbound AWS Provider Crossplane Blueprint Examples](../../README.md)
+
+
+### create 2 role, one for RDS Monitoring and one for RDS Proxy
+
+Create an IAM role and attach policy to the role (This role is required for aurora to perform monitoring)
+    
+```shell
+    aws iam create-role --role-name aurora-monitoring \
+                            --assume-role-policy-document '{
+            "Version": "2012-10-17",
+            "Statement": [
+            {
+                "Sid": "",
+                "Effect": "Allow",
+                    "Principal":
+                    {
+                    "Service": "monitoring.rds.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+
+            }
+        ]
+        }'
+ ```
+ (Attach the IAM role with AmazonRDSEnhancedMonitoringRole policy.)
+
+```shell
+    
+    aws iam attach-role-policy --role-name aurora-monitoring \
+                           --policy-arn arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole
+
+```
+
+  Create an 2nd IAM role and attach policy to the role (This role is required for aurora proxy to fetch db secrets from secret manager )
+
+```shell
+    aws iam create-role --role-name rds-proxy \
+                        --assume-role-policy-document '{    
+        "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Sid": "",
+            "Effect": "Allow",
+                "Principal":
+                {
+                "Service": "rds.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+
+        }
+    ]
+    }'
+```
+
+ ```shell
+
+    aws iam put-role-policy --role-name rds-proxy \
+                            --policy-name rds-proxy-policy \  
+                            --policy-document '{
+                            "Version": "2012-10-17",
+                            "Statement": [
+                                {
+                                    "Sid": "getsm",
+                                    "Effect": "Allow",
+                                    "Action": "secretsmanager:GetSecretValue",
+                                    "Resource": "*"
+                                },
+                                {
+                                    "Sid": "kmsdecrypt",
+                                    "Effect": "Allow",
+                                    "Action": "kms:Decrypt",
+                                    "Resource": "*",
+                                    "Condition": {
+                                        "StringEquals": {
+                                            "kms:ViaService": "secretsmanager.us-east-1.amazonaws.com"
```

Comment:
Is it guaranteed to be in `us-east-1`?

Reply:
Thats need to updated accordingly based on the region where we create the RDS.

Reply:
I will update that accordingly.

Reply:
updated